x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -198,10 +198,12 @@ export const InnerFieldItem = function InnerFieldItem(props: FieldItemProps) {
               className={`lnsFieldItem__info ${infoIsOpen ? 'lnsFieldItem__info-isOpen' : ''}`}
               data-test-subj={`lnsFieldListPanelField-${field.name}`}
               onClick={() => {
-                togglePopover();
+                if (exists) {
+                  togglePopover();
```

Comment:
The previous behavior here was confusing, but actually sort of useful, where the click state could be used to debug issues with the `exists` logic. I don't think it needs to be changed here.